### PR TITLE
fix: ios dashboard icon

### PR DESCRIPTION
**Pull request by SunriseFox**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -198,7 +198,7 @@ function Dashboard() {
...
...kv1to5E1WOm1ue9AOLLj8ZFCydl1UkLPqbcg/gySKhZxy7Fl09Gkx3Vxw9av1+r1pxvfn5caryO0jxNUKhh67yt4/r9ii+DBTXLQ3y5A+JLPcZOLFKFmFxbtuvhbEd8GQyGet6S8tpXa68dJwT6EF9Oa2rU6n234ssnb2lIdQjd9k4o3+QVZvSYNtRR2mDOciI1Rt4bw0EX9CC+dKXkBVCrGeutqQqrLHgbwD/Doy1SC4ix24UiG/TaNY2oWuHkOoDvgTGiJuyKHk7r/92MT2RqgGKyL/61VgB4AUsttWUq0xrR5Tbvjh5DjWK8I9Yp6ISxwylUDZm7oO/yi00QU70kF32X38yP5opqxQh6n+eOAKBThpqTrKkMTBagiHmPUSpRM0TbwV8FXbIcOYlCJVOnepsvhm3QjM9WVA8luWwH8EdDKw+pJlKh9fbiCgDoqjGf2yt6mOILgYWRywecjImtUK0UuzCwZ4MO+hxhVlSSaMr2fFT//782J1O1xNjVujFn0Zz3B0+liu2EI9ZvAOiqreHuI4XuIkema6B29pTlmBckkWog+/kBhzSgy5ZvhzPH0FPW9kTHwclbfLDSWKZqiH6B3teNNos9QxVRTR4rOAB032zEOoIdMSL9naj62dQ96FdUbMoy15y9StURI2ExQp0TXsPMugjxiu4mhqajo+lyl+HbwouoXDsz0DBlcZ/tsSi7U7/iUqOXn2kFgFf2bqPph0vguGmRC9Nz6gYXnjV0a3TFC68iJ2tRZKFMpVKk/YTXUOaEl7O1Dk6+91WR/kuJsvNFH2PL/KC55ZiRTM8Tw72ro+z1oOWVn8ZrKgFj5yzurAAAgNIst9blnOxEetJKMtODhqwJL2/2pvPuJBc8M+FiP9oxa0YNIz+2E+/sLAxtjP1yGZYbw/F8hR4jS6Z7HKMD/LyZtsgfP0CpJmdew9lJAACoxHJjsUEqZHG4otuJTGSbubs4atixAUD5TiMjOO99hm7D+FkeHLX3uNaqXPOxzqeevZPpJkrICS5vbdCABACAegxPV4sN3Ml5KnhmltixH4Uqo+4iybe5ZG96Ql6kDm9oGxTVAKAWw/djzTgE6dRMODv2pXD3+aUUSn5sc4k5TQPe0K7vsV2u2mwz0tmLe/a+LA1StFMZRg2/LQ5C4SxYa4SVAR70EcC8CJSGenDjAAAAAElFTkSuQmCC"
```
Comment (guanbinrui) [inline src/index.tsx]: Why not put it to `public/` folder?

Comment (SunriseFox) [reply src/index.tsx]: I don't think it necessary.

Comment (Jack-Works) [reply src/index.tsx]: hmmm it's not so friendly to the code editor(

Comment (guanbinrui) [reply src/index.tsx]: I also wanna remind here not use base64 image in injected components. Cause image will not reveal to user before stylesheets adopted.

Comment (SunriseFox) [reply src/index.tsx]: nothing will base on this commit and even itself will be dropped within this week... So I don't think any further processing is necessary.

**Pull request merged**